R/C-flux/cleaning.R

```diff
@@ -1,14 +1,34 @@
-# library(plyr) #why do I need it? maybe I don't
+# This script is to clean raw data from various loggers into one datafile with calculated fluxes
 library(tidyverse)
 source("https://raw.githubusercontent.com/jogaudard/common/master/fun-fluxes.R")
 library(lubridate, warn.conflicts = FALSE)
 library(broom)
+library(fs)
+library("dataDownloader")
```

Comment:
I have a load libraries file. You can source that one for the most basic packages. But this you can fix in the next round. We can discuss which packages should be there. I think the standard packages like tidyverse lubridate etc can be there and the more specialist we load separatly

Reply:
ok I'll have a look